pom.xml

```diff
@@ -88,8 +88,8 @@
         <cal-branding>Alliance</cal-branding>
         <admin-console>Admin Console</admin-console>
         <command-console>Command Console</command-console>
-        <public_url>http://localhost:8181</public_url>
-        <secure_url>https://localhost:8993</secure_url>
+        <public_url>\http://{FQDN}:{PORT}</public_url>
```

Comment:
❓ What are the initial slashes for?

Reply:
He answered this in the DDF PR: https://github.com/codice/ddf/pull/2285#discussion_r137345260.

Reply:
can't hurt to have it hear to. It disables creating a link that, by design, will be broken as it's a text placeholder for a user-defined value.